### ISSUE TITLE
[bug] 

### ISSUE THREAD

**Issue by zhaomo08** [persiliao/brew-services-manage#2]
### Describe the bug

always loading 
![image](https://github.com/user-attachments/assets/9d66aae7-f3a4-4bea-ba25-c4e8f341b63e)

### Full System information output

```text
Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: Mac15,11
      Model Number: Z1AG00054CH/A
      Chip: Apple M3 Max
      Total Number of Cores: 14 (10 performance and 4 efficiency)
      Memory: 36 GB
      System Firmware Version: 11881.61.3
      OS Loader Version: 11881.61.3
      Serial Number (system): JVQTK9GKQH
      Hardware UUID: CD3723DD-50B4-5DA2-812B-B687C4C93106
      Provisioning UDID: 00006034-001821063C78001C
      Activation Lock Status: Enabled
```

**Comment by persiliao** [comment]
Please run the brew services list

**Comment by zhaomo08** [comment]
<img width="662" alt="image" src="https://github.com/user-attachments/assets/88d502a7-34e6-4d35-8f8a-3e832db6944c" />


> Please run the brew services list

**Comment by persiliao** [comment]
Restart the app, it keeps showing Services Loading? You can take a look at the ~/Library/Logs/com.persiliao.bsm/Brew Services Manage.log log.

**Comment by zhaomo08** [comment]
> Restart the app, it keeps showing Services Loading? You can take a look at the ~/Library/Logs/com.persiliao.bsm/Brew Services Manage.log log.

24-12-29][11:52:51][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:09:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:18][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:48:18][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)

**Comment by persiliao** [comment]
Okay, I probably know why. I'll take care of it.

**Comment by persiliao** [comment]
mongodb-community@6.0 This service can't find information, did you create this one locally yourself or?

**Comment by zhaomo08** [comment]
> mongodb-community@6.0 This service can't find information, did you create this one locally yourself or?

I uninstalled mongodb-community@6.0 ,Still cannot be used.

(base) ☁  com.persiliao.bsm  brew services list
Name        Status     User  File
consul      none
jenkins-lts none
php         none
podman      none
rabbitmq    error  256 mesay ~/Library/LaunchAgents/homebrew.mxcl.rabbitmq.plist
redis       stopped    mesay ~/Library/LaunchAgents/homebrew.mxcl.redis.plist
zookeeper   none
(base) ☁  com.persiliao.bsm  cat Brew\ Services\ Manage.log
[2024-12-29][11:33:55][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-29][11:33:55][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-29][11:33:56][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-29][11:33:59][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-29][11:33:59][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-29][11:50:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][11:50:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][11:50:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-29][11:50:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-29][11:52:47][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-29][11:52:51][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-29][11:52:51][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-29][12:02:07][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:09:44][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:09:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:09:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:14:59][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:15][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][21:48:18][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:48:18][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 3075, column: 13)
[2024-12-30][21:51:20][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:51:20][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationWillTerminate`
[2024-12-30][21:51:20][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][21:51:20][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationWillTerminate`
[2024-12-30][22:00:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][22:00:45][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationDidFinishLaunching`
[2024-12-30][22:00:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][22:00:45][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationDidFinishLaunching`
[2024-12-30][22:00:46][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][22:00:46][TRACE][tao::platform_impl::platform::app_delegate] Triggered `applicationSupportsSecureRestorableState`
[2024-12-30][22:00:46][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][22:00:46][TRACE][tao::platform_impl::platform::app_delegate] Completed `applicationSupportsSecureRestorableState`
[2024-12-30][22:00:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 2693, column: 13)
[2024-12-30][22:00:48][ERROR][BrewServicesManage::app] brew info --json Failed to parse info JSON: Error("invalid type: map, expected a sequence", line: 2693, column: 13)

**Comment by persiliao** [comment]
please run 

```shell
brew info --json consul jenkins-lts php podman rabbitmq redis zookeeper
```

**Comment by zhaomo08** [comment]
> please run
> 
> ```shell
> brew info --json consul jenkins-lts php podman rabbitmq redis zookeeper
> ```

[r.json](https://github.com/user-attachments/files/18275281/r.json)

**Comment by persiliao** [comment]
It's fixed, you can now upgrade to 1.1.1

```shell
brew upgrade brew-services-manage
```

**Comment by zhaomo08** [comment]
ths,it works
